### PR TITLE
Adding table shortcode for multi-line tables

### PR DESCRIPTION
**Pull request by premkumr**
- Use `{{<table>}} ...{{</table>}}`
- Supports multi-line rows
- supports bullets, code blocks
- Needs full table structure specified in markdown

### CODE REVIEW

File: docs/content/preview/contribute/docs/docs-style.md [hunk 1]
```diff
@@ -56,7 +58,54 @@ ysqlsh> CREATE TABLE $$__banking__$$;
 
 #### YSQL and YCQL code blocks
 
-Tag YSQL code blocks as `sql`, and YCQL code blocks as `cql`. The source highlighting differs slightly between the two.
+Tag YSQL code blocks as `plpgsql`, and YCQL code blocks as `cql`. The source highlighting differs slightly between the two.
```
Comment (aishwarya24) [inline docs/content/preview/contribute/docs/docs-style.md:61]: what's the reason for tagging YSQL code blocks as `plpgsql`?

File: docs/content/preview/contribute/docs/docs-style.md [hunk 2]
```diff
@@ -56,7 +58,54 @@ ysqlsh> CREATE TABLE $$__banking__$$;
...
+
+### Tables
+
+Markdown supports [tables](https://www.markdownguide.org/extended-syntax/#tables). By design the table rows have to be in one line. This makes adding bullets and multi-line code blocks impossible in table cells. For this, use the shortcode `table`. For example,
```
Comment (aishwarya24) [inline docs/content/preview/contribute/docs/docs-style.md:65]:
```suggestion
Markdown supports [tables](https://www.markdownguide.org/extended-syntax/#tables). By design, the table rows have to be in one line making it impossible to add bullets and multi-line code blocks in table cells. To create such custom tables, use the shortcode `table` similar to the following example:
```

File: docs/content/preview/contribute/docs/docs-style.md [hunk 3]
```diff
@@ -56,7 +58,54 @@ ysqlsh> CREATE TABLE $$__banking__$$;
...
+ 1 | 2
+(1 row)
+```|
+<!-- row with tip block -->
+| {{</*warning title="Beware" */>}} start and end rows with the pipe symbol {{</*/warning*/>}} | 
+{{</*tip title="Awesome tip" */>}} Use 3 ticks for code blocks with pipe symbols {{</*/tip*/>}} |
+{{</*/table*/>}}
+```
+
+The above markdown should render as,
```
Comment (aishwarya24) [inline docs/content/preview/contribute/docs/docs-style.md:88]:
```suggestion
The above markdown should render a table as follows:
```

**Review by aishwarya24 — APPROVED** [review]
Looks great! Thanks @premkumr . This is very helpful.

File: docs/content/preview/contribute/docs/docs-style.md [hunk 4]
```diff
@@ -56,7 +58,54 @@ ysqlsh> CREATE TABLE $$__banking__$$;
 
 #### YSQL and YCQL code blocks
 
-Tag YSQL code blocks as `sql`, and YCQL code blocks as `cql`. The source highlighting differs slightly between the two.
+Tag YSQL code blocks as `plpgsql`, and YCQL code blocks as `cql`. The source highlighting differs slightly between the two.
+
+### Tables
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/docs-style.md:63]: Content more accurately goes on page https://docs.yugabyte.com/preview/contribute/docs/widgets-and-shortcodes/

File: docs/content/preview/contribute/docs/docs-style.md [hunk 5]
```diff
@@ -56,7 +58,54 @@ ysqlsh> CREATE TABLE $$__banking__$$;
...
+
+```md
+{{</*table*/>}}
+| col-1 | col-2 |
+| ----- | ----- |
+<!-- row with bullets and code block -->
+|
+- 1
+- 2
+- 3|
```
Comment (ddhodge) [inline docs/content/preview/contribute/docs/docs-style.md:75]:
```suggestion
- 3 |
```

File: docs/content/preview/contribute/docs/docs-style.md (hunk 1 above)
Comment (premkumr) [reply docs/content/preview/contribute/docs/docs-style.md:61]: `plpgsql` has features like `tablespaces` and other postgres specific features that `sql` does not have . so they will be highlighted too.

File: docs/content/preview/contribute/docs/docs-style.md (hunk 4 above)
Comment (premkumr) [reply docs/content/preview/contribute/docs/docs-style.md:63]: Let me move this to that section.

**Pull request merged**